spec/support/migration_helper.rb

```diff
@@ -103,7 +103,11 @@ def suppress_migration_messages
 
       def migrate_to(version)
         suppress_migration_messages do
-          if ActiveRecord::VERSION::STRING >= "5.2.0"
+          if ActiveRecord::VERSION::STRING >= "6.0.0"
+            migration_dir  = Rails.application.config.paths["db/migrate"]
+            migration_conn = ::ActiveRecord::Base.connection.schema_migration
+            ActiveRecord::MigrationContext.new(migration_dir, migration_conn).migrate(version)
```

Comment:
Is this also a problem on Rails 5.2?  https://github.com/ManageIQ/manageiq/pull/20784

Reply:
I puled this code from here:

https://github.com/ManageIQ/manageiq/pull/20778/commits/4d95a2ada9bac2368815669bee075a9e204e33d3

Which did have the `Rails.application.config.paths["db/migrate"]` fix from ManageIQ/manageiq#20784 already in it, and didn't bother to change it.

That said, since this is a `spec/*` file, unless we plan on running specs in 5.2 in containers, I don't see it being a problem, and the new code here was just some copy pasta.